src/decorations/intervals.jl

```diff
@@ -47,7 +47,7 @@ DecoratedInterval(I::Interval{T}, d::DECORATION) where T<:AbstractFloat =
     DecoratedInterval{T}(I, d)
 
 function DecoratedInterval(a::T, b::T, d::DECORATION) where T<:Real
-    a > b && return nai(T)
```

Comment:
the `nai(T)` does not work if `T` is rational, so e.g. `DecoratedInterval(3//1, 2//1)` throws an uncatched exception
This is also in the current master version

Reply:
that's quite beyond the scope of this PR, see #462